### ISSUE TITLE
Start Page / Release Notes - how to not pop them up every time?

### ISSUE THREAD

**Issue by cartermp** [gitpod-io/gitpod#15398]
## Is your feature request related to a problem? Please describe

Here's what I see when I create a click the gitpod button:

<img width="849" alt="image" src="https://user-images.githubusercontent.com/6309070/208200430-922fd9ed-d54d-4e65-a60d-08f311c1cfbc.png">

From this repo: https://github.com/cartermp/site

## Describe the behaviour you'd like

I'd love it if ... neither popped up, since I'm a pretty active user and not new to the VSCode experience either.

## Describe alternatives you've considered

Having a notification for the latest release notes that correspond to the month that they're for would be great. If it's mid-december then I don't think I need to see the November release notes.

**Comment by akosyakov** [comment]
> Having a notification for the latest release notes that correspond to the month that they're for would be great. If it's mid-december then I don't think I need to see the November release notes.

Changelog is released at the end of Month. Current release is November. It looks correct.

**Comment by cartermp** [comment]
> It is not correct if you see it in each new workspace. It should be shown only once per a release.

Gotcha, good to know, thanks! Just confirmed that I still see it in the specific repo I mentioned, so it's reproducible if that helps.

**Comment by filiptronicek** [comment]
Maybe this is a duplicate of https://github.com/gitpod-io/gitpod/issues/14533, I think we can leave this one open to keep track here and dig deeper into the issue. 

And @cartermp if before we fix the issue you would like to disable showing release notes, you can follow my instructions in https://github.com/gitpod-io/gitpod/issues/14533#issuecomment-1361158810.

**Comment by jeanp413** [comment]
We added `"gitpod.showReleaseNotes"`vscode setting, to enable/disable it
